src/lib/omerlo/reader/server/hooks.ts

```diff
@@ -9,21 +9,22 @@ import { refresh } from "./token";
 const handleApiProxy: Handle = async ({ event, ...tail }) => {
   event.url.host = env.PRIVATE_OMERLO_HOST;
   event.url.protocol = env.PRIVATE_OMERLO_PROTOCOL
-  event.request.headers.delete('cookie');
-  event.request.headers.set('x-omerlo-media-id', env.PRIVATE_OMERLO_MEDIA_ID);
+  event.url.port = ''; // This to prevent custom posts when working with localhost
 
   let accessToken = event.locals.accessToken;
 
   if (!accessToken) {
     accessToken = await getApplicationToken();
   }
 
-  event.request.headers.set('Authorization', `Bearer ${accessToken}`);
-
   return await fetch(event.url.toString(), {
     body: event.request.body,
     method: event.request.method,
-    headers: event.request.headers,
+    headers: {
+      'Authorization': `Bearer ${accessToken}`,
+      'Content-Type': event.request.headers.get('content-type') ?? 'application/json',
```

Comment:
the fetcher from the request must already have setted the content type. I don't think this is needed.

I can see why you need that but at the same time, this could cause issues for client with custom proxy configuration. For now I think you should put a note to describe why you do that and what does it fix.


Something like

```js
// NOTE: ....
```

Otherwise the next dev could break it trying to fix something else